docs/operations/faq/zone_aware_routing.rst

```diff
@@ -0,0 +1,47 @@
+.. _common_configuration_zone_aware_routing:
```

Comment:
Overall, this document has broken flow. You start out well with "there are several steps... but then it gets a little choppy. Overall, I would break your sections up as you have, the add a sentence explaining what you're doing in the section before numbering the steps in each section.